brave/brave-core
# Remove Ready to Start earning modal and incorporate into Connect Wallet modal

Resolves https://github.com/brave/brave-browser/issues/29448



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## UI:

### Desktop
![Screenshot 2023-04-06 090850](https://user-images.githubusercontent.com/5033691/230411005-f6d521ad-6f43-4c90-b0fc-e3aee5293596.png)

### Mobile
![Screenshot 2023-04-06 093031](https://user-images.githubusercontent.com/5033691/230411016-eb834a7b-57e4-4554-be27-9006f8a001ee.png)

## Test Plan:

- Clean profile
- Enable Rewards
- Click "Connect" button in Rewards panel
- Ensure that Connect modal looks/behaves as described in issue
